task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter A Number :");
+            int index = int.Parse(Console.ReadLine());
```

Comment:
Bad name (index) for variable. What does it mean? Index of what? Would be better name it like inputNumber or smth like this.